components/ai_chat/resources/page/components/model_selector/index.tsx

```diff
@@ -103,11 +126,9 @@ export function ModelSelector() {
             <div className={styles.menuText}>
               <div>{model.displayName}</div>
               <p className={styles.modelSubtitle}>
-                {getLocale(
-                  `CHAT_UI_${model.key
-                    .toUpperCase()
-                    .replaceAll('-', '_')}_SUBTITLE`,
-                )}
+                {model.key === 'chat-automatic'
+                  ? getLocale(S.CHAT_UI_CHAT_AUTOMATIC_SUBTITLE)
+                  : getModelCapabilities(model.capabilities)}
```

Comment:
What if we just have a `string model_description` field in `struct Model` and, if this has a value, then we use that instead of the capabilities list? Just trying to reduce and remove any hardcoding of model keys in the frontend. Same with icon by the way that I notice we have `getModelIcon` for: we could just have `model.icon_name`. This preserves the ability for all this data to come from the server and also to be rendered elsewhere.

Reply:
Actually better yet, we can check if a model has `capabilities`, if not we fallback to the `description`. Then there is no need for an additional field in the `struct Model`. What do you think?